vllm/engine/arg_utils.py

```diff
@@ -183,4 +184,8 @@ def add_cli_args(
         parser.add_argument('--disable-log-requests',
                             action='store_true',
                             help='disable logging requests')
+        parser.add_argument('--max_log_len',
+                            type=int,
+                            default=None,
+                            help='max prompt/token being printed in log')
```

Comment:
Nit: 
```suggestion
        parser.add_argument('--max-log-tokens',
                            type=int,
                            default=None,
                            help='max number of prompt tokens being printed in log')
```
Please rename `max_log_len` to `max_log_tokens` in other places. Thanks!

Reply:
Done for the rename